### ISSUE TITLE
Regression: attr() function does not validate

### ISSUE THREAD

**Issue by eligrey** [w3c/css-validator#24]
I believe that this is a regression, as the CSS 2.1 attr() function used to validate. Some recent change in 2015 has stopped it from validating. Neither CSS 2.1-style `attr(attribute)` nor CSS 3.0-style `attr(attribute <type>)` validate correctly.

Reduced testcase:

    .test::before {
       content: attr(title);
    }

**Comment by cvrebert** [comment]
:+1:

**Comment by codeitdude** [comment]
bump

**Comment by jamesharvard** [comment]
Further test case, which seems to trip up the parser more violently than @eligrey's test case:

```css
div::after {
    content: 'Title attribute is "' attr(title) '".';
}
```

**Comment by theartofweb** [comment]
Another common case used in print style sheets:

`  a[href ^= 'http']:link:after, a[href ^= 'http']:visited:after {
    content: " [" attr(href) "]";
    font-weight: normal;
    font-size: 80%;
    color: blue;
  }
`

**Comment by ylafon** [comment]
There are two issues here, attr() support (which is in the 'attr' branch currently), and the content: property support which is outdated to a point that it can't work with attr(), it will be fixed soon.

**Comment by ylafon** [comment]
fixed by #50
